src/main/java/net/minestom/server/instance/Instance.java

```diff
@@ -750,8 +750,8 @@ public EntityTracker getEntityTracker() {
      *
      * @return the instance unique id
      */
-    public @NotNull UUID getUniqueId() {
-        return uniqueId;
+    public @NotNull UUID getUuid() {
```

Comment:
I'm down, but I would prefer to keep the old method deprecated in cases like this where it is trivial to maintain backwards compatibility for a grace period.

Reply:
I added it back but deprecated it

Reply:
Please add also forRemoval also into deprecation annotation and add Javadoc with replacement!